odometry/src/odometry_node.cpp

```diff
@@ -0,0 +1,42 @@
+#include "ros/ros.h"
```

Comment:
nitpick - use angle brackets to `#include` ROS library code. helps to distinguish between code you wrote and code other people wrote